salt/_modules/kubernetes.py

```diff
@@ -1569,7 +1569,7 @@ def __dict_to_service_spec(spec):
         if key == 'ports':
             spec_obj.ports = []
             for port in value:
-                kube_port = kubernetes.client.V1ServicePort()
+                kube_port = kubernetes.client.V1ServicePort(port=port)
```

Comment:
this dirty fix should be removed and we shall apply the fix from the saltstack repo @NicolasT  ?

Reply:
Which fix would that be?

Reply:
Just cherry picked all your commit @sayf-eddine-scality should I remove this one ?

Reply:
did not push any fix for this issue yet but I created a tracking ticket #697

Reply:
basically this fix will fail once we specify `target_port` or `source_port` in the state